i_scene_cp77_gltf/main/common.py

```diff
@@ -155,17 +155,18 @@ def bsdf_socket_names():
         socket_names['Emission']= 'Emission Color'
     return socket_names    
 
-def json_ver_validate( json_data):
```

Comment:
if we're changing this about maybe we should make it return a string rather than a bool, and just define the error messages once here and send it back?

Reply:
Why not have it return a bool but also have it handle the error messaging?

Reply:
that works too.